pandas/tseries/offsets.py

```diff
@@ -115,25 +115,10 @@ def wrapper(self, other):
     return wrapper
 
 
-def _is_normalized(dt):
-    if (dt.hour != 0 or dt.minute != 0 or dt.second != 0 or
-            dt.microsecond != 0 or getattr(dt, 'nanosecond', 0) != 0):
-        return False
-    return True
-
 # ---------------------------------------------------------------------
 # DateOffset
 
 
```

Comment:
can you add a note in other api changes, that CacheableOffset & Weekday are no longer public (prob not ever used  publicly but who knowns).